docs/Linux_Install_Guide.rst

```diff
@@ -38,9 +38,9 @@ As a general rule, 8GB of system memory is required for a full hipTensor build.
 
 GPU Support
 ^^^^^^^^^^^
-AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a as 'gfx9'
+AMD CDNA class GPU featuring matrix core support: gfx908, gfx90a, gfx940, gfx941, gfx942 as 'gfx9'
```

Comment:
Please ensure consistency with what's published in the GPU Support matrix on the portal.